cfme/tests/control/test_alerts.py

```diff
@@ -120,6 +120,17 @@ def setup_for_alerts(request, alerts, event=None, vm_name=None, provider=None):
 
 
 @pytest.yield_fixture(scope="module")
+def set_performance_capture_threshold():
+    yaml = store.current_appliance.get_yaml_config()
```

Comment:
REQUIRED: sorry, I didn't pay enough attention on that. Please replace store.current_appliance with appliance fixture.